Helper/Data.php

```diff
@@ -1901,4 +1901,21 @@ public function getPspReferenceSearchUrl($pspReference, $liveEnvironment)
         );
 
     }
+
+    /**
+     * Gives back CountryCode configuration values
+     *
+     * @param null $storeId
+     * @return mixed
+     */
+    public function getAdyenCountryCodeConfigData($storeId = null)
+    {
+        $shopperLocale = trim($this->_adyenHelper->getAdyenHppConfigData('shopper_locale', $storeId));
+        $countryCode = trim($this->_adyenHelper->getAdyenHppConfigData('country_code', $storeId));
+        if ($countryCode == false && $shopperLocale == 1) {
```

Comment:
@AlexandrosMor from [this line](https://github.com/Adyen/adyen-magento2/blob/102e262d6bc7c73c25c90c76354d3a18d9db0b92/Gateway/Command/PayByMailCommand.php#L155) it looks like country_code could be empty instead of FALSE. Have you tested generating a PayByMail link or PaymentMethods call with the new shopper_country=1 and shopper_country=0? I think this IF would never evaluate to true and the country_code is not sent